### PR TITLE
Removed comment on tests for add Followers, and add Following.

### PR DESCRIPTION
**Pull request by drincon2**
Removed commented code in the act and assert sections of the tests for adding Followers and Following.

### CODE REVIEW

File: Fakebook.Profile/Fakebook.Profile.UnitTests/DomainTests/ProfileModelTests.cs [hunk 1]
```diff
@@ -283,5 +285,41 @@ public void EarlierTodayIsValid()
...
+        {
+            // arrange
+            DomainProfile profile = new("emailtest@gmail.com", "Bob", "Fields");
+            string followEmail = "testermail@gmail.com";
+
+            // act
+            profile.AddFollow(followerEmail);
+
+            // assert
+            Assert.Contains(followEmail, profile.FollowEmails);
```
Comment (tdunbar123) [inline Fakebook.Profile/Fakebook.Profile.UnitTests/DomainTests/ProfileModelTests.cs:321]: Change profile.FollowEmails to profile.FollowingEmails to be more accurate.

File: Fakebook.Profile/Fakebook.Profile.UnitTests/DomainTests/ProfileModelTests.cs [hunk 2]
```diff
@@ -283,5 +285,41 @@ public void EarlierTodayIsValid()
...
+
+        /// <summary>
+        /// Check that following emails have the correct format.
+        /// </summary>
+        [Fact]
+        public void TestAddFollowingEmail()
+        {
+            // arrange
+            DomainProfile profile = new("emailtest@gmail.com", "Bob", "Fields");
+            string followEmail = "testermail@gmail.com";
```
Comment (tdunbar123) [inline Fakebook.Profile/Fakebook.Profile.UnitTests/DomainTests/ProfileModelTests.cs:315]: Change followEmail to followingEmail to be more accurate

File: Fakebook.Profile/Fakebook.Profile.UnitTests/DomainTests/ProfileModelTests.cs [hunk 3]
```diff
@@ -283,5 +285,41 @@ public void EarlierTodayIsValid()
...
+
+            // act
+            profile.AddFollow(followerEmail);
```
Comment (tdunbar123) [inline Fakebook.Profile/Fakebook.Profile.UnitTests/DomainTests/ProfileModelTests.cs:318]: Make sure this argument name is matching the variable name you are creating above!

**Review by tdunbar123 — CHANGES_REQUESTED** [review]

**Review by escalonn — CHANGES_REQUESTED** [review]
even with tdd-fail, it should at least compile. so you should add the methods to DomainProfile (editor lightbulb can do it for you) but leave them throwing NotImplementedException.

you should have tests for invalid cases as well as valid cases.

that said, i'm a little unsure about this design. it shouldn't be possible to add A as a follower of B without also adding B as a follow of A at the same time. that fact should be managed somewhere in the domain project, maybe by a FollowService. this PR doesn't need to handle that, but just remember it.

but, well, in the database, they are one and the same thing hmm. maybe only one of these two methods should exist. i guess i'll leave it to the judgment of you guys

File: Fakebook.Profile/Fakebook.Profile.Domain/DomainProfile.cs [hunk 4]
```diff
@@ -12,6 +13,10 @@ public class DomainProfile
     {
         //[anything]@[anything].[anything]
         private string _email;
+
+        public List<string> FollowerEmails;
+        public List<string> FollowingEmails;
+
```
Comment (escalonn) [inline Fakebook.Profile/Fakebook.Profile.Domain/DomainProfile.cs:19]: public fields are bad, address code smells

**Review by escalonn — CHANGES_REQUESTED** [review]

File: Fakebook.Profile/Fakebook.Profile.Domain/DomainProfile.cs (hunk 4 above)
Comment (escalonn) [reply Fakebook.Profile/Fakebook.Profile.Domain/DomainProfile.cs:30]: having full properties here isn't really needed in this PR. you could have left it as auto-properties, or as properties that throw NotImplementedException (like the IDE lightbulb will generate for you if you try to reference them while they don't exist at all). but it's good enough.

also, when you make changes to address previous review comments, click "resolve conversation" on them, after leaving a reply indicating how you addressed it if it's not obvious.

File: Fakebook.Profile/Fakebook.Profile.Domain/DomainProfile.cs [hunk 5]
```diff
@@ -212,6 +238,9 @@ public DomainProfile(string email, string firstName, string lastName)
             FirstName = firstName;
             LastName = lastName;
             ProfilePictureUrl = new Uri(ProfileConfiguration.DefaultUri);
+            FollowerEmails = new List<string>();
+            FollowingEmails = new List<string>();
```
Comment (escalonn) [inline Fakebook.Profile/Fakebook.Profile.Domain/DomainProfile.cs:242]: FYI this code also has no reason to be in this PR. with TDD phase 1, literally the only code you add that's not tests is the minimum needed to get the tests compiling and failing. but i'll let it slide this time.

**Review by escalonn — APPROVED** [review]
